flax/experimental/nnx/tests/test_rngs.py

```diff
@@ -181,3 +181,34 @@ def test_rng_stream_pytree(self):
     assert set(keys.keys()) == set(keys2.keys())
     assert set(keys.splits.keys()) == set(keys2.splits.keys())
     assert set(keys.broadcasts.keys()) == set(keys2.broadcasts.keys())
+
+  def test_jit_updates(self):
+    class Foo(nnx.Module):
+      def __init__(self, not_rngs):
+        rngs = not_rngs
+        self.linear = nnx.Linear(2, 2, rngs=rngs)
```

Comment:
```suggestion
      def __init__(self, rngs):
        self.linear = nnx.Linear(2, 2, rngs=rngs)
```
Small nit: What is the purpose of calling the arg `not_rngs` when it's reassigned to `rngs` and passed in as an rng key to an `nnx.Module` anyway?